assets/xml/objects/object_du.xml

```diff
@@ -0,0 +1,82 @@
+<Root>
+    <File Name="object_du" Segment="6">
+        <!-- Darunia's skeleton -->
+        <Skeleton Name="gDaruniaSkel" Type="Flex" LimbType="Standard" Offset="0x11CA8"/>
+
+        <Animation Name="gDaruniaIdleAnim" Offset="0x6EB0"/>
+
+        <Animation Name="gDaruniaItemGiveAnim" Offset="0x800"/>
+        <Animation Name="gDaruniaItemGiveIdleAnim" Offset="0xD00"/>
+
+        <!-- Used in the Goron's Ruby Cutscene -->
+        <Animation Name="gDaruniaHitLinkAnim" Offset="0x1D70"/>
+        <Animation Name="gDaruniaHitBreastAnim" Offset="0x2374"/>
+        <Animation Name="gDaruniaStandUpAfterFallingAnim" Offset="0x288C"/>
+
+        <!-- Animations of Darunia dancing the Saria's Song -->
+
+        <!-- He opens and closes his arms -->
+        <Animation Name="gDaruniaDancingLoop1Anim" Offset="0x2D94"/>
+        <!-- I don't understand this one -->
+        <Animation Name="gDaruniaDancingLoop2Anim" Offset="0x3D48"/>
+        <!-- Dances side to side -->
+        <Animation Name="gDaruniaDancingLoop3Anim" Offset="0x4C04"/>
+        <!-- I don't know. He thinks he is a chicken maybe (?) -->
+        <Animation Name="gDaruniaDancingLoop4Anim" Offset="0x4ED8"/>
+        <!-- Seems to go from from dancing to idle -->
+        <Animation Name="gDaruniaDancingEndAnim" Offset="0x41F4"/>
+
+        <!-- Link played a song which isn't the Saria's Song -->
+        <Animation Name="gDaruniaWrongSongAnim" Offset="0x3A30"/>
+        <Animation Name="gDaruniaWrongSongEndAnim" Offset="0x46F4"/>
+
+        <!-- Cutscene in the Chamber of Sages, after killing Ganon -->
+        <Animation Name="gDaruniaSageFormationAnim" Offset="0x12014"/>
+
+        <!-- Cutscene in credits. Saria is in Darunia's shoulders -->
+        <Animation Name="gDaruniaLookingUpToSariaAnim" Offset="0x5458"/>
+        <Animation Name="gDaruniaCreditsHitBreastAnim" Offset="0x6104"/>
+        <Animation Name="gDaruniaCreditsIdleAnim" Offset="0x67CC"/>
+
+
+        <!-- Darunia's eyes textures -->
+        <!-- TODO: Rename darunia_eye_half1 and darunia_eye_half2 to something more clear -->
+        <Texture Name="gDaruniaEyeOpenTex" OutName="darunia_eye_open" Format="ci8" Width="32" Height="32" Offset="0x8080"/>
+        <Texture Name="gDaruniaEyeHalf1Tex" OutName="darunia_eye_half1" Format="ci8" Width="32" Height="32" Offset="0x8480"/>
+        <Texture Name="gDaruniaEyeShutTex" OutName="darunia_eye_shut" Format="ci8" Width="32" Height="32" Offset="0x8880"/>
+        <Texture Name="gDaruniaEyeHalf2Tex" OutName="darunia_eye_half2" Format="ci8" Width="32" Height="32" Offset="0xA540"/>
+
+        <!-- Darunia's mouth textures -->
+        <Texture Name="gDaruniaMouthSeriousTex" OutName="darunia_mouth_serious" Format="ci8" Width="64" Height="32" Offset="0x8C80"/>
+        <Texture Name="gDaruniaMouthTeethsTex" OutName="darunia_mouth_teeths" Format="ci8" Width="64" Height="32" Offset="0x9D40"/>
```

Comment:
"teeths" -> "teeth"
though id try to describe the facial expression if possible, grinning maybe?